root/defaults/nginx/site-confs/default.conf.sample

```diff
@@ -9,7 +9,7 @@ server {
 
     server_name _;
 
-    root /config/www/xbackbone;
+    root /app/xbackbone;
```

Comment:
Can this be `/app/www/public` so that it is in-line with other apps (since https://github.com/linuxserver/docker-baseimage-alpine-nginx/pull/84 )? This would apply to the init script as well, but could be changed via a mass search/replace.